### PR TITLE
[Contracts 1.1] Prospective Release of Contracts 1.1

### PR DESCRIPTION
**Pull request by keefertaylor**
Contracts 1.1 is a maintenance release coupled with prioritization of the Liquidity Pool for Liquidations.

This release contains:
#1 - Add ability to set dev fund split
#3 - Decompose `updateContracts` entrypoint
#4 - Remove `ovenMax` parameter
#5 - Decompose `updateParams` entrypoint
#8 - Prioritize Liquidation Pool for Liquidations
#9 - Deployment and Testing scripts
#16 - Updating API from [comments on KIP-006](https://discuss.kolibri.finance/t/kip-006-prioritize-the-liquidation-pool-in-liquidations/58/17)
#17 - Remove accidentally committed SmartPy artifacts
#18 - Fix nits from PR reviewer

This PR effectively implements and provides migration scripts for 
[KIP-007: Enhancements to the Minter Contract API](https://discuss.kolibri.finance/t/kip-005-enhancements-to-the-minter-contract-api/57)
[KIP-006: Prioritize the Liquidation Pool in Liquidations](https://discuss.kolibri.finance/t/kip-006-prioritize-the-liquidation-pool-in-liquidations/58)

### CODE REVIEW

File: migrations/contracts-1.1/src/flows/generate-governance-lambda.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import * as config from '../config'
+import { fetchFromCache } from '../utils'
+import { compileLambda } from '../utils/compile-break-glass-lambda'
+
+const main = async () => {
+    // Contracts
+    const ovenFactoryContract = config.coreContracts.OVEN_FACTORY
+    const ovenProxyContract = config.coreContracts.OVEN_PROXY
+    const tokenContract = config.coreContracts.TOKEN
```
Comment (Fitblip) [inline migrations/contracts-1.1/src/flows/generate-governance-lambda.ts:9]: Should these all have `!` at the end to ensure they're never null?

File: migrations/contracts-1.1/src/storage-wrappers/minter-contract.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+          (nat %interestIndex) 
+          (timestamp %lastInterestIndexUpdateTime)
+        )
+      )
+    ) 
+    (Pair 
+      (Pair
+        (nat %liquidationFeePercent) 
+        (Pair 
+          (address $liquidityPoolAddress)
```
Comment (Fitblip) [inline migrations/contracts-1.1/src/storage-wrappers/minter-contract.ts:66]: I think this should be `%liquidityPoolAddress` (though it's correctly replacing currently below, so don't forget to update that too!)

**Review by Fitblip — APPROVED** [review]
Groovy, this looks good to me! Huge thanks for banging this out \o/

**Comment by keefertaylor** [comment]
I've resolved your nits in #18 and merged to this branch.

**Comment by keefertaylor** [comment]
I have tested all changes on Sandbox net according to the runbook in migrations/contracts-1.1/README.md and they pass all tests.

**Pull request merged**